src/Components/test/testassets/BasicTestApp/RouterTest/Links.razor

```diff
@@ -1,5 +1,5 @@
 @using Microsoft.AspNetCore.Components.Routing
-@inject Microsoft.AspNetCore.Components.Services.IUriHelper uriHelper
+@inject Microsoft.AspNetCore.Components.IUriHelper uriHelper
```

Comment:
This can be simplified since `.Components` is always in scope

Reply:
Yeah, I'm doing a pass on top of this to clean up any other possible rename (and just get all of them done in one go)